src/common/gui/CAboutBox.cpp

```diff
@@ -152,7 +160,19 @@ CAboutBox::onMouseDown(CPoint& where,
    if (!(button & kLButton))
       return kMouseEventNotHandled;
 
-   boxHide();
+   //char debug[128];
```

Comment:
please don't leave this in the code. thanks!

Reply:
oh drat, forgot to remove that.

Reply:
Updated in cf82b26